packages/strapi-plugin-i18n/admin/src/components/DropdownIndicator/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Flex } from '@buffetjs/core';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const Wrapper = styled(Flex)`
+  height: 100%;
+  width: 32px;
+  justify-content: space-around;
+  background: #fafafb;
+  > svg {
+    align-self: center;
+    font-size: 11px;
+    color: #b3b5b9;
+  }
+`;
+
+const DropdownIndicator = ({ selectProps: { menuIsOpen } }) => {
```

Comment:
This one already exists in strapi-helper, I don't think we need to duplicate it

Reply:
Well it does not have the correct UI we need this one so its similar to the one used in the other react select.